### DISCUSSION TITLE
Setup prettier like how eslint via eslint-custom-config has been configured

### DISCUSSION THREAD

**Discussion by joggienl** [vercel/turborepo#4075] [Help] [answered]
Hello!

Out of curiosity of what others think, would you setup prettier similar to 'eslint-custom-config' from the example and thus leveraging caching? Whats your take on this?

**Comment by anthonyshew** [comment, marked as answer]
My personal opinion is that Prettier tends to be the fastest task you'll have by far, even if you only execute it once in the root instead of fanning it out to a package command. Often, the configuration overhead isn't worth it.

That said, as your monorepo grows, you may find that your CI is waiting for Prettier. If you ever come across this situation, you could then go for the approach you're describing.

So, to give the developer's answer: It depends. 😄
